### ISSUE TITLE
Please add partition functions for 12C12C and 12C13C on ExoMol website

### ISSUE THREAD

**Issue by RobertAReedUSAF** [ExoMol/exomol.com#24]
Partition function for 12C12C is already available elsewhere & simply needs to be posted on ExoMol site. But I don't know what is available for the isotopic variant 12C13C.

**Comment by Trovemaster** [comment]
Dear RobertAReedUSAF, 

The partition functions of all three isotopologues of C2 should be already available,  see e.g. 
[https://exomol.com/data/molecules/C2/12C-13C/8states/](https://exomol.com/data/molecules/C2/12C-13C/8states/) 
[https://exomol.com/data/molecules/C2/12C2/8states/](https://exomol.com/data/molecules/C2/12C2/8states/)

Do you mean something else? 

Cheers,
Sergey

**Comment by RobertAReedUSAF** [comment]
Thank you Sergey,
Not all molecules on the ExoMol website have partition functions. For example, CO has a partition function but C2 does not. If I am reading correctly, pyExoCross will compute partition functions from the line lists. But I’m hoping that someone has already computed partition functions for C2 (and isotopologues) and is willing to post those results on the ExoMol site for the rest of us.
Thank you, Bob 😊

From: Sergey Yurchenko ***@***.***>
Sent: Thursday, January 25, 2024 3:34 PM
To: ExoMol/exomol.com ***@***.***>
Cc: REED, ROBERT A CTR USAF AFMC AEDC/TST ***@***.***>; Author ***@***.***>
Subject: [Non-DoD Source] Re: [ExoMol/exomol.com] Please add partition functions for 12C12C and 12C13C on ExoMol website (Issue #24)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Dear RobertAReedUSAF,

The partition functions of all three isotopologues of C2 should be already available, see e.g.
https://exomol.com/data/molecules/C2/12C-13C/8states/
https://exomol.com/data/molecules/C2/12C2/8states/

Do you mean something else?

Cheers,
Sergey

**Comment by Trovemaster** [comment]
Dear Bob, 

Sorry, but I am still not sure I understand why you cannot find partition functions of C2. Can you please give more details, prinscreen? 
Here is for example a precomputed partition function of 12C13C: 
https://exomol.com/db/C2/12C-13C/8states/12C-13C__8states.pf
Here is a partition function of 12C12C: 
https://exomol.com/db/C2/12C2/8states/12C2__8states.pf 

Do you mean something else? 

Cheers, 
Sergey

**Comment by RobertAReedUSAF** [comment]
Yes Sergey! This is exactly what I need. Sorry, I don’t know why I missed them.
Thank you very much! Bob 😊

From: Sergey Yurchenko ***@***.***>
Sent: Saturday, January 27, 2024 2:23 PM
To: ExoMol/exomol.com ***@***.***>
Cc: REED, ROBERT A CTR USAF AFMC AEDC/TST ***@***.***>; Author ***@***.***>
Subject: [Non-DoD Source] Re: [ExoMol/exomol.com] Please add partition functions for 12C12C and 12C13C on ExoMol website (Issue #24)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Dear Bob,

Sorry, but I am still not sure I understand why you cannot find partition functions of C2. Can you please give more details, prinscreen?
Here is for example a precomputed partition function of 12C13C:
https://exomol.com/db/C2/12C-13C/8states/12C-13C__8states.pf
Here is a partition function of 12C12C:
https://exomol.com/db/C2/12C2/8states/12C2__8states.pf

Do you mean something else?

Cheers,
Sergey
